rm_vk/rm_vk/FriendsTableViewController/FriendsTableViewController.swift

```diff
@@ -0,0 +1,120 @@
+        static let friendFiveName = "Maggie"
+        static let friendDescription = "Springfield"
+    }
+
+    // MARK: - Private Properties
+
+    private let users = [
+        User(
+            avatarImageName: Constants.friendOneName.lowercased(),
+            name: Constants.friendOneName,
+            description: Constants.friendDescription
+        ),
+        User(
+            avatarImageName: Constants.friendTwoName.lowercased(),
+            name: Constants.friendTwoName,
+            description: Constants.friendDescription
+        ),
+        User(
+            avatarImageName: Constants.friendThreeName.lowercased(),
+            name: Constants.friendThreeName,
+            description: nil
+        ),
+        User(
+            avatarImageName: Constants.friendFourName.lowercased(),
+            name: Constants.friendFourName,
+            description: Constants.friendDescription
+        ),
+        User(
+            avatarImageName: Constants.friendFiveName.lowercased(),
+            name: Constants.friendFiveName,
+            description: nil
+        ),
+        User(
+            avatarImageName: Constants.friendOneName.lowercased(),
+            name: Constants.friendOneName,
+            description: Constants.friendDescription
+        ),
+        User(
+            avatarImageName: Constants.friendTwoName.lowercased(),
+            name: Constants.friendTwoName,
+            description: Constants.friendDescription
+        ),
+        User(
+            avatarImageName: Constants.friendThreeName.lowercased(),
+            name: Constants.friendThreeName,
+            description: nil
+        ),
+        User(
+            avatarImageName: Constants.friendFourName.lowercased(),
+            name: Constants.friendFourName,
+            description: Constants.friendDescription
+        ),
+        User(
+            avatarImageName: Constants.friendFiveName.lowercased(),
+            name: Constants.friendFiveName,
+            description: nil
+        )
+    ]
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        registerCell()
+    }
+
+    // MARK: - UITableViewDataSource
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        users.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(
+            withIdentifier: Constants.cellIdentifier, for: indexPath
+        ) as? MainTableViewCell else { return UITableViewCell() }
+        cell.configureUser(model: users[indexPath.row])
+        return cell
+    }
+
+    // MARK: - UITableViewDelegate
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        performSegue(withIdentifier: Constants.segueIdentifier, sender: users[indexPath.row].avatarImageName)
```

Comment:
```suggestion
        performSegue(withIdentifier: Constants.segueIdentifier, sender: users[indexPath.row].avatarImageName)
```
сторибордом же делаем, и переходы сторибордом